src/Covid19Dashboard/index.jsx

```diff
@@ -444,6 +459,7 @@ Covid19Dashboard.defaultProps = {
     country: {}, state: {}, county: {}, last_updated: '',
   },
   jhuJsonByTimeLatest: { il_county_list: {}, last_updated: '' },
+  vaccinesByCountyByDate: { il_county_list: {}, last_updated: '' },
```

Comment:
nitpicking: is this really the shape of `vaccinesByCountyByDate` object?

if not, better change to reflct what will be the actual shape of that object

Reply:
should i put the total? see below change or leave it off, otherwise the shape is the same

Reply:
yeah I think it make sense to me to have that `total` in the default object, since you were expecting that field in code